module/VuFind/src/VuFind/ILS/Driver/Folio.php

```diff
@@ -1180,6 +1180,7 @@ public function patronLogin($username, $password)
             'firstname' => $profile->personal->firstName ?? null,
             'lastname' => $profile->personal->lastName ?? null,
             'email' => $profile->personal->email ?? null,
+            'addressTypeIds' => array_map(fn ($address) => $address->addressTypeId, $profile->personal->addresses),
```

Comment:
Should you add `?? []` after `$profile->personal->addresses` just in case the element is missing?

Reply:
Yup, done.